src/main/java/com/conveyal/r5/analyst/Grid.java

```diff
@@ -102,6 +102,12 @@
     /** The data values for each pixel within this grid. */
     public final double[][] grid;
 
+    /** Maximum area allowed for the bounding box of an uploaded shapefile -- large enough for New York State.  */
+    private static final double MAX_BOUNDING_BOX_AREA_SQ_KM = 250000;
```

Comment:
Both the Netherlands and Switzerland are around 40k km2, and their bounding boxes could be up to twice that big. So this is big enough to contain shapefiles for those locations. I did misread this at first at 25k km2. Tip: Java lets you put spaces in number, so you can write `MAX_BOUNDING_BOX_AREA_SQ_KM = 250 000;`. We should make this standard for constants >= 10k in our Java code. I'll update the style guide.

Reply:
👍

Reply:
Just curious @ansoncfit, didn't your IDE show an error with the space in the constant? I guess you would expect Travis CI to tell you if there was a compile error but it's failing due to a misconfiguration - I'll fix that.

Reply:
It did show an error.  I thought it was just a style setting I hadn't changed, but I should have paid more attention to it.